docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -119,6 +134,9 @@ autonomy mint agent [OPTIONS] PACKAGE_PATH
 `--owner OWNER_ADDRESS`
 : Owner address of the package.
 
+`--update TOKEN_ID`
+: Update the existing mint with the latest package hash.
```

Comment:
same suggestion of previous comment

Reply:
Addressed [c241c59](https://github.com/valory-xyz/open-autonomy/pull/2050/commits/c241c59d71cdbcfc957398e1e6eb69eaef5daab1)